src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
+function getAssetPaths() {
+    if (assetPaths === null) {
+        let data = fs.readFileSync(path.resolve(__dirname, 'assets.json'));
+        assetPaths = JSON.parse(data.toString());
+    }
+
+    return assetPaths;
+}
 
```

Comment:
Do you need it to be a function, or could we just always initialize the paths from the very start.  I don't see why not.  Basically, what I mean is:

    const assetPaths = getAssetPaths();

And where getAssetPaths is a singly-called helper that does

    const data = fs.readFileSync(path.resolve(__dirname, 'assets.json'));
    return JSON.parse(data.toString());